packages/playground/src/views/nodes.vue

```diff
@@ -487,21 +511,18 @@ export default {
     });
 
     const loading = ref<boolean>(true);
-    const nodes = ref<GridNode[]>([]);
+    const _nodes = ref<GridNode[]>([]);
+
+    const sortItem = ref<{ title: string; icon: string; value: (nodes: GridNode[]) => GridNode[] }>();
```

Comment:
It is recommended to use `reactive` for objects and `ref` for primitives.

Reply:
nothing special about reactive.

from dev io website
The significant difference between ref() and reactive() is that the ref() function allows us to declare reactive state for primitives and objects, while reactive() only declares reactive state for objects.

from vue docs (it's recommended to use `ref`)
https://vuejs.org/guide/essentials/reactivity-fundamentals#ref